app/Http/Controllers/Entity/EditingController.php

```diff
@@ -6,7 +6,12 @@
 
 use App\Http\Controllers\Controller;
 use App\Models\Entity;
+use App\Models\EntityNote;
+use App\Models\Campaign;
+use App\Models\QuestElement;
+use App\Models\TimelineElement;
 use App\Services\Entity\MultiEditingService;
+use Ramsey\Uuid\Type\Time;
 
 class EditingController extends Controller
```

Comment:
Since this isn't anymore limited to just Entities, we can move the namespace and route names